web/src/lib/Components/Table.svelte

```diff
@@ -13,4 +13,4 @@
   })
 </script>
 
-<div id={tableId} class="h-3/4 w-3/4 ag-theme-alpine-dark" />
+<div id={tableId} class="h-3/4 w-9/10 ag-theme-alpine-dark" />
```

Comment:
Why this change?

Reply:
In the future, we could look at tools like percy which will show an image diff ...

Reply:
Yes, previously (at 75%), after fixing the overflow issue the table was not filling the space entirely, at 90% it utilizes the space and leaves an equal margin on both sides. 

At 75%
![image](https://user-images.githubusercontent.com/55191777/179217564-7421365c-d1f8-4d31-ac5b-f8088a13d053.png)

At 90%
![image](https://user-images.githubusercontent.com/55191777/179217714-8fd942ac-8824-4be1-a453-b72aae1a38a2.png)